src/operations/create_collection.ts

```diff
@@ -17,6 +17,7 @@ import { Aspect, defineAspects } from './operation';
 const ILLEGAL_COMMAND_FIELDS = new Set([
   'w',
   'wtimeout',
+  'timeoutMS',
```

Comment:
Added here as the benchmarks were adding the `timeoutMS` to the actual command being sent to the server when creating a collection